sql/localsql/nipost/poet_registration.go

```diff
@@ -47,29 +72,54 @@ func ClearPoetRegistrations(db sql.Executor, nodeID types.NodeID) error {
 	return nil
 }
 
-func PoetRegistrations(db sql.Executor, nodeID types.NodeID) ([]PoETRegistration, error) {
+func PoetRegistrations(db sql.Executor, nodeIDs ...types.NodeID) ([]PoETRegistration, error) {
```

Comment:
I don't think querying multiple nodeIDs in a singe statement is faster than doing the same in multiple queries. It can still be a single transaction by passing one as the `db` parameter to this function.

Changing `PoetRegistrations` to query for multiple identities just makes the function more complicated when the caller can do the same thing easily without changing this query.